CMakeLists.txt

```diff
@@ -309,7 +310,6 @@ set(TEST_FILENAMES
   DPPP/test/unit/tScaleData.cc
   DPPP/test/unit/tStationAdder.cc
   DDECal/test/unit/tRotationConstraint.cc
-  DPPP_Interpolate/test/tInterpolateStep.cc
```

Comment:
I think it is better to leave the test in de DPPP_Interpolate directory since we do the same for DDECal tests.